### PR TITLE
Added funkquist-et-al-kr2024 reference

### PR DESCRIPTION
**Pull request by Martin36**
@jendrikseipp @dgnad

### CODE REVIEW

File: tests/literatur-sorted.bib
Comment (jendrikseipp) [inline tests/literatur-sorted.bib:1]: That file should not be committed to the repo.

File: literatur.bib [hunk 1]
```diff
@@ -6159,6 +6159,13 @@ @InProceedings{fuentetaja-et-al-ipc2018
   crossref =     "ipc2018"
 }
 
+% TODO: Add pages
```
Comment (jendrikseipp) [inline literatur.bib]:
```suggestion
% TODO(Martin): add pages.
```

File: literatur.bib [hunk 2]
```diff
@@ -6159,6 +6159,13 @@ @InProceedings{fuentetaja-et-al-ipc2018
...
+@InProceedings{funkquist-et-al-kr2024,
+  author =       "Martin Funkquist and Simon Ståhlberg and Hector Geffner",
```
Comment (jendrikseipp) [inline literatur.bib]: The å needs to be escaped, see other entries for Simon.

**Review by jendrikseipp — CHANGES_REQUESTED** [review]

File: literatur.bib (hunk 1 above)
Comment (Martin36) [reply literatur.bib]: Done

File: literatur.bib (hunk 2 above)
Comment (Martin36) [reply literatur.bib]: Done

**Pull request merged**